src/main/resources/application.properties

```diff
@@ -30,7 +30,10 @@ spring.liquibase.change-log=classpath:/db/changelog/db.changelog-master.xml
 definition.cache.max-idle.secs=${DEFINITION_CACHE_MAX_IDLE_SEC:259200}
 
 #ttl for the latest version objects cached
-definition.cache.latest-version-ttl.secs=${DEFINITION_CACHE_LATEST_VERSION_TTL_SEC:30}
+definition.cache.latest-version-ttl=${DEFINITION_CACHE_LATEST_VERSION_TTL_SEC:30}
```

Comment:
I think we should keep the `.sec` suffix to avoid confusion

Reply:
We can later on refactor to include the unit of measurement in the value, like 5sec. But for now, let the unit of measurement stay in the describing the property, rather than in the property name itself.